static/templates/user_profile_modal.hbs

```diff
@@ -63,5 +63,23 @@
             </div>
             {{/each}}
         </div>
+        <div id="total-list-sections">
+            <div id="groups-list-section" class="list-section">
+                <h4>Groups Part of</h4>
+                <ul>
+                    {{#each groups_of_user}}
+                    <li>{{this.name}}</li>
+                    {{/each}}
+                </ul>
+            </div>
+            <div id="streams-list-section" class="list-section">
+                <h4>Streams Part of</h4>
+                <ul>
+                    {{#each user_streams}}
+                    <li>{{this.name}}</li>
+                    {{/each}}
+                </ul>
+            </div>
```

Comment:
@Abhirup-99 can you split out the showing of streams in separate commit. Also, see https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages  for improving your commit message.

Reply:
Can you suggest why the check failed? I didn't change anything in python.

Reply:
I have updated the code please review.